Mathlib/RingTheory/Ideal/Height.lean

```diff
@@ -0,0 +1,118 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jiedong Jiang, Jingting Wang, Andrew Yang
+-/
+import Mathlib.Order.KrullDimension
+import Mathlib.RingTheory.Ideal.MinimalPrime
+import Mathlib.RingTheory.PrimeSpectrum
```

Comment:
This file has recently moved. Please merge master and replace this with `Mathlib.RingTheory.Spectrum.Prime.Defs` (or `.Basic`)